# Hare_rabbit_microbiome

Illumina_BLAST_CLI.ipynb: The fasta sequences from QIIME2 post DADA2 stage was downloaded and analysised through BLASTn command line interface. The output file along with the "feature table" from QIIME2 are analysed to create a taxonomy map.

Merging_Illumina_Nanopore_data: Nanopore dataset and Illumina dataset are merged within QIIME2 for comparing 

Nanopore_16S_BLASTouput_for_QIIME2.ipynb: Each fasta file per sample was run through BLASTn command line interface. The output files were imported into jupyter notebook for processing to make it QIIME2 compatible.

merging_sample_data.ipynb: Merging all filtered BLAST output files into one to create a feature table for importing into QIIME2

QIIME2_Illumina_MiSeq_16S_data: Processing Illumina samples through QIIME2

QIIME2_Nanopore_16S_data: Processing Nanopore samples through QIIME2

